templates/splash/layout-splashpage.html

```diff
@@ -6,18 +6,19 @@
 		<title>{{ page.title }}</title>
 		<meta content="width=device-width, initial-scale=1" name="viewport"/>
```

Comment:
After further discussions and research, you would need to add the following reference to the Dublin Core namespace right under the `title` tag. What's important here is that it is included before your first instance of a `meta` with `dcterms`. No need for the `profile` attribute on the `head` which would be just for XHTML page and not for HTML.
 tag.
```suggestion
		<link rel="schema.dcterms" href="http://purl.org/dc/terms/" />
		<meta content="width=device-width, initial-scale=1" name="viewport"/>
```